src/changelog/comment_changes.rs

```diff
@@ -296,15 +303,28 @@ impl Logic {
                 }
 
                 if let Ok(commit) = repository.find_commit(oid) {
-                  if let Some(message) = if self.cli.body {
-                    commit.body()
-                  } else {
-                    commit.summary()
-                  } {
+                  if let Some(message) = commit.message() {
+                    let (summary, body) =
+                      message.split_once('\n').unwrap_or((message, ""));
+
                     if let Some((category, change)) =
-                      self.harvest_message(message)
+                      self.harvest_message(if self.cli.body {
+                        body.trim()
+                      } else {
+                        summary.trim()
+                      })
                     {
                       Self::insert(&mut result, category, change);
+                    } else if self.cli.force {
+                      if let Some((category, change)) =
+                        self.harvest_message(if self.cli.body {
+                          summary.trim()
+                        } else {
+                          body.trim()
+                        })
+                      {
+                        Self::insert(&mut result, category, change);
+                      }
```

Comment:
If the first search in the preferred message part did not succeed, the opposite message part will be searched *if* the force mode is active.  The results of the second search will be applied in any case:

- both queries fail → no fragment
- second query succeeds → fragment from second query
- second query succeeds due to fallback category → fragment from second query